Logger/LogFactory.cs

```diff
@@ -1,11 +1,16 @@
-﻿namespace Logger
+﻿using System;
+
+namespace Logger
 {
     public class LogFactory
     {
+        private string logFilePath = null;
```

Comment:
Fields should be underscore PascalCase. In C#, there is no need to initialize variables to their default values.
```suggestion
        private string _LogFilePath = null;
```